ext/datasource/k8s/deploy/crds/datasource.sentinel.com_hotspotrules_crd.yaml

```diff
@@ -0,0 +1,125 @@
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: HotspotRulesSpec defines the desired state of HotspotRules
+          properties:
+            rules:
+              description: 'rules is the hot spot(frequency) rule list'
+              items:
+                properties:
+                  resource:
+                    type: string
+                    minLength: 1
+                    maxLength: 100
+                  metricType:
+                    type: string
+                    enum:
+                    - Concurrency
+                    - QPS
+                  controlBehavior:
+                    type: string
+                    enum:
+                    - Reject
+                    - Throttling
+                  paramIndex:
+                    description: ParamIndex is the index in context arguments list.
+                    type: integer
+                    minimum: 0
+                  threshold:
+                    format: number
+                    type: float
+                  maxQueueingTimeMs:
+                    description: MaxQueueingTimeMs is the max queueing time in Throttling ControlBehavior
+                    format: int64
+                    type: integer
+                    minimum: 0
+                  burstCount:
+                    format: int64
+                    type: integer
+                    minimum: 0
+                  durationInSec:
+                    format: int64
+                    type: integer
+                    minimum: 0
+                  paramsMaxCapacity:
+                    format: int64
+                    type: integer
+                    minimum: 1
+                  specificItems:
+                    description: specific item list
+                    items:
+                      description: 'specific item'
+                      properties:
+                        valKind:
+                          format: string
+                          enum:
+                          - Int
+                          - String
+                          - Bool
+                          - Float64
+                        valStr:
+                          type: string
+                          minLength: 1
+                          maxLength: 100
+                        specificThreshold:
+                          format: int64
+                          type: integer
+                          minimum: 0
+                      required:
+                      - valKind
+                      - valStr
+                      - specificThreshold
+                      type: object
+                    type: array
+                required:
+                - resource
+                - metricType
+                - controlBehavior
+                - paramIndex
+                - threshold
+                - maxQueueingTimeMs
```

Comment:
Please also simplify the required list here.

Reply:
Okay